dags/process_elife_zip_dag.py

```diff
@@ -90,8 +95,38 @@ def prepare_jats_xml_for_libero(**context):
 
     s3 = get_aws_connection('s3')
     response = s3.get_object(Bucket=DESTINATION_BUCKET, Key=xml_path)
-    xml = etree.parse(BytesIO(response['Body'].read()))
-    return etree.tostring(xml)
+    article_xml = etree.parse(BytesIO(response['Body'].read()))
+
+    # get article id
+    article_id = article_xml.xpath(
+        '/article/front/article-meta/article-id[@pub-id-type="publisher-id"]'
+    )[0].text
+
+    # add jats prefix to jats tags
+    for element in article_xml.iter():
+        if not element.prefix:
+            element.tag = '{http://jats.nlm.nih.gov}%s' % element.tag
+
+    # add xml:base attribute to article element
+    key = Path(xml_path).parent.stem
+    root = article_xml.getroot()
+    root.set(
+        '{%s}base' % XML_NAMESPACE,
+        '%s/%s/%s' % (ENDPOINT_URL, DESTINATION_BUCKET, key)
```

Comment:
Here's where it gets complicated. After expansion, the files are in the expanded bucket which at the moment is configured to be publicly accessible so that a `content-store` can pull the files without authentication. It's difficult to add authentication without being too much AWS-specific, as the `content-store` would have to generate a signature to add to requests in order to see the files.
So at the moment the files will be publicly accessible as soon as you get here, if you know the file names. This is just for seconds in this workflow, as than the `content-store` makes them public anyway not having yet a concept of `preview` article not ready for publication. Medium-term will need to do something about it, could be as simple as expanding the files into an unguessable folder:
```
https://unstable-expanded.s3.amazon.aws.com/fdb8d906-77cb-470d-9090-40a046e81bf9/elife-666-fig1.jpg
```
We probably don't want to implement this yet, just a possibility (of many).